corehq/pillows/xform.py

```diff
@@ -135,7 +135,6 @@ def transform_xform_for_elasticsearch(doc_dict):
 
 def get_xform_to_elasticsearch_pillow(pillow_id='XFormToElasticsearchPillow', num_processes=1,
                                       process_num=0, **kwargs):
-    assert pillow_id == 'XFormToElasticsearchPillow', 'Pillow ID is not allowed to change'
```

Comment:
seems odd that this assert is here given that you can pass it in as an argument

Reply:
I chose not to dig too deeply. I assume that there was some reasonable migration going on and that asserts were there to ensure no configs changed. There are similar asserts on other pillows